contracts/p1/Distributor.sol

```diff
@@ -160,6 +194,13 @@ contract DistributorP1 is ComponentP1, IDistributor {
             revTotals.rTokenTotal += share.rTokenDist;
             revTotals.rsrTotal += share.rsrDist;
         }
+
+        // DAO Fee
+        (, uint256 feeNumerator) = main.daoFeeRegistry().getFeeDetails(address(rToken));
+        revTotals.rsrTotal += uint24(
+            (feeNumerator * uint256(revTotals.rTokenTotal + revTotals.rsrTotal)) /
+                (1e4 - feeNumerator)
```

Comment:
should read the 1e4 out from `daoFeeRegistry()`, or could add it as a return value to `getFeeDetails()`. but shouldn't just in-line the 1e4 imo